pkg/controller/controller_instance_test.go

```diff
@@ -454,6 +480,10 @@ func TestReconcileInstanceWithProvisionFailure(t *testing.T) {
 		t.Fatalf("Expected failed condition to be set")
 	}
 
+	if updatedInstance.Status.Conditions[1].Type != v1alpha1.InstanceConditionReady && updatedInstance.Status.Conditions[1].Status != v1alpha1.ConditionFalse {
+		t.Fatalf("Expected failed condition to be set")
```

Comment:
comment doesn't seem to match the logic here, and I don't think we need this check, since `assertInstanceReadyFalse` is called above.